source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ImageRenderer.java

```diff
@@ -136,35 +136,24 @@ else if (imageSize == ImageSize.Large)
 
     private static void setImageSize(Context context, ImageView imageView, ImageSize imageSize, ImageSizesConfig imageSizesConfig, boolean isInImageSet) {
         imageView.setScaleType(ImageView.ScaleType.CENTER);
-        if (imageSize == ImageSize.Stretch)
+        if (imageSize != ImageSize.Small && imageSize != ImageSize.Medium && imageSize != ImageSize.Large)
```

Comment:
NIt: Readability & code maintenance: I would extract a quick function out of this for readability to make it clear what exactly it is that this check is doing - IsAllowedImageSizeValue - and then add a quick comment on there explaining the policy decision to default to medium.